oneanddone/tests/functional/conftest.py

```diff
@@ -14,7 +16,18 @@ def capabilities(capabilities):
 
 @pytest.fixture
 def persona_test_user():
-    return requests.get('http://personatestuser.org/email').json()
+    max_retries = 5
+    attempt = 1
+    while attempt <= max_retries:
+        user_info = requests.get('http://personatestuser.org/email').json()
```

Comment:
This will raise a `ValueError` if the response is not valid JSON. We probably want to retry in that scenario.